pkg/cmd/run.go

```diff
@@ -128,10 +139,18 @@ func NewRunCommand(factory cmdutil.Factory, streams genericclioptions.IOStreams)
 		},
 	}
 
+	// flags for existing usage
 	cmd.Flags().StringVarP(&o.container, "container", "c", o.container, "Specify the container")
-	cmd.Flags().BoolVarP(&o.attach, "attach", "a", o.attach, "Whether or not to attach to the trace program once it is created")
 	cmd.Flags().StringVarP(&o.eval, "eval", "e", o.eval, "Literal string to be evaluated as a bpftrace program")
-	cmd.Flags().StringVarP(&o.program, "filename", "f", o.program, "File containing a bpftrace program")
+	cmd.Flags().StringVarP(&o.filename, "filename", "f", o.filename, "File containing a bpftrace program")
```

Comment:
If i'm reading this right these  flags will be removed in the future?

If yes then ...

I'd recommend at a minimum marking them as deprecated.

Reply:
Not decided yet, but if we do then I will mark them all as deprecated.

Reply:
After looking at future use cases this one should stay alongside `--program`.